interfaces/matlab_experimental/Phases/Solution.m

```diff
@@ -0,0 +1,91 @@
+classdef Solution < handle & ThermoPhase & Kinetics & Transport
```

Comment:
After the merge of #1448, there are several significant simplifications possible. Once you rebase to the current `main`, you can load a `Solution` directly using the following new `clib` functions:
```c
int soln = soln_newSolution(infile, name, transport);
int thermo = soln_thermo(soln);
int kin = soln_kinetics(soln);
int trans = soln_transport(soln);
```
I.e. you no longer have to load phase and kinetics information separately.

On a separate note, the `Solution.m` file should probably reside in a new folder `base` in side the `matlab_experimental` folder.

Reply:
Thanks a lot! This is a very helpful change to the currently very awkward way of constructing the `Solution` object in Matlab.